crates/hyperswitch_domain_models/src/payment_methods.rs

```diff
@@ -77,9 +78,11 @@ pub struct PaymentMethod {
     pub merchant_id: common_utils::id_type::MerchantId,
     pub created_at: PrimitiveDateTime,
     pub last_modified: PrimitiveDateTime,
-    pub payment_method: Option<storage_enums::PaymentMethod>,
-    pub payment_method_type: Option<storage_enums::PaymentMethodType>,
-    pub payment_method_data: OptionalEncryptableValue,
+    pub payment_method_type: Option<storage_enums::PaymentMethod>,
+    pub payment_method_subtype: Option<storage_enums::PaymentMethodType>,
+    pub payment_method_data: Option<
```

Comment:
Would it be better to use a type alias here?

Reply:
I did add a type alias for it, I'm taking it up in my next PR.